frontend/src/components/header/MenuOverlay.js

```diff
@@ -675,8 +681,18 @@ class MenuOverlay extends Component {
                     />
                   ))}
 
-                {queriedResults.length === 0 && query !== '' && (
-                  <span>There are no results</span>
+                {queriedResults.length === 0 &&
+                  query !== '' &&
+                  !pendingQuery && (
+                    <span>
+                      {counterpart.translate('window.noResults.caption')}
+                    </span>
+                  )}
+
+                {pendingQuery && queriedResults.length === 0 && (
```

Comment:
IMHO the `queriedResults.length === 0` is not needed

Reply:
In this case yes it is not needed. Thanks